src/nvim/change.c

```diff
@@ -1611,6 +1611,7 @@ int open_line(
     }
     ml_replace(curwin->w_cursor.lnum, p_extra, true);
     changed_bytes(curwin->w_cursor.lnum, 0);
+    // TODO(vigoux): extmark_splice_cols here??
```

Comment:
So, to splice or not to splice ?

Reply:
lets revisit this when we write the test for this case.